src/ui/Facet/BreadcrumbValueElement.ts

```diff
@@ -100,5 +101,12 @@ export class BreadcrumbValueElement {
         facetTitle: this.facet.options.title
       })
     );
+
+    this.focusOnContainer();
+  }
+
+  private focusOnContainer() {
+    const breadcrumb = first(this.facet.searchInterface.getComponents<Component>('Breadcrumb'));
```

Comment:
For some reason, this doesn't seem to work with TalkBack on Android. The same issue happens for both Facet and DynamicFacet (which seem to produce different breadcrumbs).
Steps to reproduce:
1. Select 2 values in Facet A
2. Select 1 value in Facet B
3. Close the facet popup and turn on TalkBack
4. Select the second value from Breadcrumb A then double-tap. The Breadcrumb A is selected. 
![image](https://user-images.githubusercontent.com/54454747/91881982-54a72380-ec50-11ea-89aa-de79a73b4f09.png)
5. Swipe right to select the next element. The label of Breadcrumb A's first value is selected.
6. Swipe right again to select the clear button of Breadcrumb A's only value and double-tap. Nothing is focused anymore.

Reply:
Mehhhhhh ...

Reply:
Since those are fix requested by a client for desktop version, I think it's more logical to have target that platform only for now.

We can revisit later if needed.